dev/how-to/maintain/tidb-in-kubernetes/delete-cluster.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 销毁 Kubernetes 上的 TiDB 集群
+category: how-to
+---
+
+# 销毁 TiDB 集群
+
+> **警告：**
+>
+> 下列命令会彻底删除数据，务必考虑清楚再执行。
+
+要销毁 TiDB 集群，执行以下命令：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+helm delete ${releaseName} --purge
+```
+
+上述命令只是删除运行的 Pod，数据仍然会保留。如果你不再需要那些数据，可以通过下面命令清除数据：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl delete pvc -n ${namespace} -l app.kubernetes.io/instance=${releaseName},app.kubernetes.io/managed-by=tidb-operator && \
```

Comment:
Since the second command is already too long, I suggest separating these two commands.